tests/test_admin/test_view.py

```diff
@@ -158,7 +158,7 @@ def setUp(self):
         super(BaseTestEnterpriseCustomerManageLearnersView, self).setUp()
         self.user = UserFactory.create(is_staff=True, is_active=True, id=1)
         self.worker_user = UserFactory.create(
-            username=settings.ENTERPRISE_SERVICE_WORKER_USERNAME, is_staff=True, is_active=True
+            username=settings.ENTERPRISE_SERVICE_WORKER_USERNAME, is_staff=True, is_active=True, id=1000
```

Comment:
why we need id?

Reply:
Tests are failing randomly in `test_view.py` due to `IntegrityErrors`, As per my understanding this is happening due to manually specifying `id` in lot `UserFactory` creation instances.

Reply:
yeah but if we have 999 users already?